third_party/bigframes_vendored/ibis/expr/operations/analytic.py

```diff
@@ -2,22 +2,24 @@
 
 from __future__ import annotations
 
-from ibis.expr.operations.analytic import Analytic
+import ibis.expr.datatypes as dt
+import ibis.expr.operations.analytic as ibis_ops_analytic
+import ibis.expr.operations.core as ibis_ops_core
 import ibis.expr.rules as rlz
 
 
-class FirstNonNullValue(Analytic):
+class FirstNonNullValue(ibis_ops_analytic.Analytic):
     """Retrieve the first element."""
 
-    arg = rlz.column(rlz.any)
-    output_dtype = rlz.dtype_like("arg")
+    arg: ibis_ops_core.Column[dt.Any]
```

Comment:
You can just simply use `ops.Column` without the parameter, it means the same thing as `ops.Column[dt.Any]`